pkg/config/config.toml.nextgen.example

```diff
@@ -0,0 +1,477 @@
+# TiDB Configuration.
```

Comment:
please add `keyspace` config, it must be set in nextgen

Reply:
Done